conans/test/integration/toolchains/intel/_base.py

```diff
@@ -0,0 +1,56 @@
+import os
+import platform
+import textwrap
+import unittest
+
+from conans.client.tools.intel import intel_installation_path
+from conans.errors import ConanException
+from conans.test.utils.tools import TestClient
+
+
+class BaseIntelTestCase(unittest.TestCase):
```

Comment:
Check the new ``pytest.mark`` annotations for tests and use them.

Reply:
are they documented somewhere? e.g. what does `@pytest.mark.tool_compiler` mean - does it mean working compiler is required to run test?

Reply:
Not documented yet, pretty new, just have a look to the other tests.
Yes, you should use ``@pytest.mark.tool_icc`` for example for intel, and if using CMake, or MSBuild, use ``@pytest.mark.tool_cmake`` too.

Reply:
added pytest annotations